src/App.js

```diff
@@ -1,10 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
 import React from 'react';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import About from './components/About';
+import Quotes from './components/Quotes';
+import './App.css';
 
 const App = () => (
-  <div>
-    <Calculator />
-  </div>
+  <>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Calculator />} />
+      <Route path="/quote" element={<Quotes />} />
+      <Route path="/about" element={<About />} />
```

Comment:
- [ ] Good job creating the full math magician web site 👏 👏 . You created 3 pages however the three pages you created does not match the wireframe provided here. You should have the `home page` that match the home page in the wireframe, the `calcultor` page that match the `calculator` page in the wireframe, and the same for `quote`. 😄 , Kindle that a look at the wireframe then update your website 👍 

#### Wireframe

![Capture1](https://github.com/Esmatullah12/math-magicians/assets/80458199/f5878375-c1d1-40a1-a5e5-710713875d32)

### Your website

![Capture](https://github.com/Esmatullah12/math-magicians/assets/80458199/510f1773-4b64-4673-ac44-f496251f703d)